README.md

```diff
@@ -1,3 +1,21 @@
 # hackmud_wiki
 
 [![License: CC BY-NC-SA 4.0](https://img.shields.io/badge/License-CC_BY--NC--SA_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc-sa/4.0/)
+
+## Setting up a local environment
+
+### Prerequisites
+
+- [pnpm](https://pnpm.io/installation)
+
+### Installation
+
+1. Clone the repository
+2. Run `pnpm install` in the root directory
+3. Set your Node.js runtime to version >= `20.x` (as specified in `.package.json`)
+    - You can achieve this by running `pnpm env use 20.x --global`
+
+### Running the development server
+
+1. Run `pnpm dev` in the root directory
```

Comment:
There is no `dev` script defined in `package.json`. I think this is supposed to be `pnpm start`.

Reply:
You are correct, my mistake